src/bokeh/models/tools.py

```diff
@@ -1606,7 +1606,7 @@ class PolyDrawTool(PolyTool, Drag, Tap):
       to add each subsequent vertex, to finalize the draw action double tap to
       insert the final vertex or press the <<esc> key.
```

Comment:
```suggestion
      insert the final vertex or press the <<esc>> key.
```

Here is an inconsistency, too.